test/com/gfredericks/test/chuck/properties_test.cljc

```diff
@@ -0,0 +1,49 @@
+(ns com.gfredericks.test.chuck.properties-test
+  #?(:cljs (:require-macros [com.gfredericks.test.chuck.properties.macros :as prop']
+                            [com.gfredericks.test.chuck.generators :as gen']))
+  #?(:clj  (:require [clojure.test :refer :all]
+                     [clojure.test.check :as t.c]
+                     [clojure.test.check.generators :as gen]
+                     [clojure.test.check.clojure-test :refer [defspec]]
+                     [com.gfredericks.test.chuck.properties :as prop'])
+     :cljs (:require [cljs.test :refer-macros [run-tests deftest is]]
+                     [cljs.test.check :as t.c]
+                     [cljs.test.check.generators :as gen]
+                     [cljs.test.check.properties :include-macros true]
+                     [cljs.test.check.cljs-test :refer-macros [defspec]])))
```

Comment:
Okay there's a branch up called `cljs-namespace-renaming` that you ought to be able to play with. I'm going to try it on one of my own projects and release an RC if it works